File: Programacion/Python/Ej_Repaso/ej1.py
n1 = int(input("Dime un número: "))
n2 = int(input("Dime otro número: "))
n3 = int(input("Dime otro número: "))

if n1 < n2 and n1 < n3:
    print(f'El número menor es {n1}')
elif n1 > n2 and n2 < n3:
    print(f'El número menor es {n2}')
else:
    if n3 < n2 and n3 < n1:
        print(f'El número menor es {n3}')

File: DAM_2º-año_1º/Programación/UD7 - Kotlin Ej1/src/Ej1.kt
/**
 * 1. Solicita dos números al usuario y muestra el mayor.
 * **/
fun main(){
    print("Dime un número: ")
    val n1 = readLine()?.toInt();

    print("Dime otro número: ")
    val n2 = readLine()?.toInt();

    if (n1 != null) {
        if (n1> n2!!){
            println("$n1 Es el número más alto");
        }else{
            println("$n2 Es el número más alto");
        }
    }
}